Service.rb

```diff
@@ -0,0 +1,90 @@
+require_relative 'Car.rb'
+
+class Service 
+  
+  OPEN_SERVICE = 8
+  CLOSE_SERVICE = 20
+  CAR_CLEANING_TIME = 120
+	OPEN_SATURDAY = 10
+	CLOSE_SATURDAY = 16
+  
+  def initialize
+    @worker1 = Array.new
+    @worker2 = Array.new
+  end
+
+  def pickup(car)
+    if @worker1.empty?
+      car.car_time = car.start_time
+      calculate_pickup(car)
+      @worker1.push(car)
+      return
+    elsif @worker2.empty?
+      car.car_time = car.start_time
+      calculate_pickup(car)
+      @worker2.push(car)
+			return
+    end
+      
+    if @worker1.last.pickup_time < car.start_time
+    	car.car_time = car.start_time
+      calculate_pickup(car)
+      @worker1.push(car)
+			return
+    end
+    if @worker2.last.pickup_time < car.start_time
+      car.car_time = car.start_time
+      calculate_pickup(car)
+      @worker2.push(car)
+			return
+    end
+      
+    if @worker1.last.pickup_time < @worker2.last.pickup_time
+    	car.car_time = @worker1.last.pickup_time 
+      calculate_pickup(car)
+      @worker1.push(car)
+			return
+    else
+      car.car_time = @worker2.last.pickup_time
+      calculate_pickup(car)
+      @worker2.push(car)
+		end
```

Comment:
the indentation seems a little bit off here too and in other lines throughout this file